basic/stam/StamExAnswer.py

```diff
@@ -0,0 +1,43 @@
+__author__ = "Keren"
+
+FIRST_NAME = "Keren"
+LAST_NAME = "Guez"
+
+
+def add_vals(line):
+    """
```

Comment:
look at chapter 3.8 how the docs should look like
https://github.com/google/styleguide/blob/gh-pages/pyguide.md#38-comments-and-docstrings